src/mono/wasm/runtime/loader/exit.ts

```diff
@@ -98,6 +129,17 @@ function set_exit_code_and_quit_now(exit_code: number, reason?: any): void {
     }
 
     appendElementOnExit(exit_code);
+    if (!ABORT && !runtimeHelpers.runtimeReady) abort_startup(reason, false, false);
+    if (runtimeHelpers.mono_wasm_exit) {
+        try {
+            runtimeHelpers.mono_wasm_exit(exit_code);
+        }
+        catch (expected) {
+            if (expected && runtimeHelpers.ExitStatus && !(expected instanceof runtimeHelpers.ExitStatus)) {
```

Comment:
is swallowing this going to cause any problems? shouldn't we re-throw it at the end of this function?

Reply:
It would eventually throw at line 146 or 151. But PR doesn't change how `set_exit_code_and_quit_now` throws.
We swallow it here because most callers of don't need it. Those which do, could throw after the call and some of them do.